### PR TITLE
Bug 1154209 - [l10n][Camera]Arabic:The checkboxes will be overlapped with the text in the Website Permission view, r=alive

### PR DESCRIPTION
**Pull request by gasolin**
...h the text in the Website Permission view, r=alive

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5083187c44f7dbe38833a47c3dba6b5be336610c

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d39bcb44356b67b1e07d549bb78989efc9c32b16

File: apps/system/style/permission_manager/permission_manager.css [hunk 1]
```diff
@@ -113,6 +113,10 @@
   padding-left: 7rem;
 }
 
+html[dir="rtl"] #permission-screen[data-type] #permission-message {
```
Comment (alivedise) [inline apps/system/style/permission_manager/permission_manager.css]: There should be no rtl specification when you use -moz-padding-start...

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d07136cbc8b2ea8d5f0a2c3d52e235f708459771

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=caec5234ced31f776f427e1e6349df3e3667ef76

File: apps/system/style/permission_manager/permission_manager.css [hunk 2]
```diff
@@ -122,7 +126,10 @@
 #permission-device-selector {
   font-size: 1.8rem;
   font-weight: 300;
-  padding: 3rem 0 2rem 1.2rem;
+  padding-top: 3rem;
+  padding-bottom: 1.5rem;
```
Comment (gasolin) [inline apps/system/style/permission_manager/permission_manager.css]: li.device-cell will eat the 0.5rem

File: apps/system/style/permission_manager/permission_manager.css [hunk 3]
```diff
@@ -137,6 +144,8 @@
 #permission-devices li.device-cell {
   position: relative;
   list-style-type: none;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
```
Comment (gasolin) [inline apps/system/style/permission_manager/permission_manager.css]: UI designer prefer 10px space between checkbox

File: apps/system/style/permission_manager/permission_manager.css [hunk 4]
```diff
@@ -157,7 +166,7 @@
 
 #permission-devices input[type="checkbox"] + span {
   position: absolute;
-  top: 0.3rem;
+  top: 1.3rem;
```
Comment (gasolin) [inline apps/system/style/permission_manager/permission_manager.css]: align checkbox with text

File: apps/system/style/permission_manager/permission_manager.css [hunk 5]
```diff
@@ -58,6 +58,10 @@
   word-wrap: break-word;
 }
 
+html[dir="rtl"] #permission-message {
+  background-position: right 0 top 50%;
```
Comment (gasolin) [inline apps/system/style/permission_manager/permission_manager.css]: when RTL, the background image need to be align right :sweat_smile:

File: apps/system/style/permission_manager/permission_manager.css [hunk 6]
```diff
@@ -110,7 +114,7 @@
 }
 
 #permission-screen[data-type] #permission-message {
-  padding-left: 7rem;
+  -moz-padding-start: 7rem;
```
Comment (gasolin) [inline apps/system/style/permission_manager/permission_manager.css]: therefore the message could be adjust based on background position

**Pull request merged**